freelance-agreement.md

```diff
@@ -0,0 +1,135 @@
+
+## What are we agreeing to?
+
+As an individual contributor, you are contributing **_original works_** of code
+to the open source codebase ("**the codebase**") which forms the "**dwyl app**",
+maintained **dwyl ltd** and hosted on GitHub at: https://github.com/dwyl
+
+As a **contributor**, you hereby confirm that:
+
+1. The contributions you make to the codebase are **_your original work_**
+(_never copied from somewhere else_)
+2. By committing your code into a dwyl-owned repository, **you grant worldwide, and irrevocable but _non-exclusive_ copyright with no further consideration**
+3. This is an open source project, so in addition to **dwyl ltd**. using your (and all of our) contributions, the codebase may also be used by other people (3rd parties) without further consent or consideration.
++ Your contribution **does not grant you a _claim_ to any earnings** (past, present or future) of **dwyl Ltd**. or any other individuals or companies that use your code.
++ You **do not have any _obligation_ to provide support for your code**, however you are encouraged to remain a contributor/supporter for as long you find it personally beneficial/rewarding;
++ If you use the original works you contribute to the dwyl codebase in future projects we ask that you provide **attribution** back to the project where you originally made the contribution.
+
+We (_dwyl Ltd_.) agree to the following:
+
+a. **ensure that you are listed as a contributor** where ever your **original work** has been used; in addition to the permanent contribution on GitHub we will list you on the dwyl website "about" page with a link to your preferred online profile(s). If you chose to exclude yourself from the list of contributors on the dwyl website, please notify us by raising an issue on: https://github.com/dwyl/dwyl.github.io/issues  
+b. **You can use your contribution for any other purpose** including other open source projects, provided you communicate clearly with the owners/maintainers of those projects where/when you produced the "**original work**" and provide **attribution** to that effect.
+
+
+
+## Tracking Work
+
+You acknowledge and agree that:
+
+1. **Work is tracked _exclusively_ on GitHub issues**.
++ Each task should have its own GitHub issue and be clearly related to a _component_ of the **dwyl app**;
++ If you discuss something related to a story/feature/task _verbally_, always ensure it is captured in a comment in the GitHub issue for posterity; your memory is not as good as you think it is and the person reviewing the feature/code might not be in the same country as you! As much detail/context as possible in the issue is always better.
++ Tasks _**must**_ **be _estimated_** (_in minutes - not "points" - by everyone present and an **average recorded** as the estimate_) **_before_ being _started_**.
```

Comment:
"everyone present" won't work in the context of remote freelancers. "by at least 3 other dwyl contributors"?